### PR TITLE
1.7.6.2 release note

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.2"
+subtitle: "Maintenance version of the 1.7.6.x branch"
+date:   2019-08-26 16:15:00
```
Comment (atomiix) [inline news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]: C'est la date de la release 1.7.6.1 ça je crois non ?

Comment (ttoine) [reply news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]:
```suggestion
date:   2019-11-28 16:15:00
```

Comment (matks) [reply news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]:
```suggestion
date:   2019-11-28 16:15:00
```

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## Main fixes
+Below are listed some of the main regression issues we fixed in this version :
+
+- An infinite loop within the checkout page when two specific cart rules were applied to an order ([#15574](https://github.com/PrestaShop/PrestaShop/issues/15574))
+- One bug impacting contributor and partner payment modules due to amount paid no longer validated when creating an order ([#15834](https://github.com/PrestaShop/PrestaShop/issues/15834))
+- Three bugs related to the CLDR implementation in the 1.7.6.0 which were provoking errors in both suppliers and orders pages after deleting a currency ([#15376](https://github.com/PrestaShop/PrestaShop/issues/15376)), ([#15486](https://github.com/PrestaShop/PrestaShop/issues/15486)) and ([#15956](https://github.com/PrestaShop/PrestaShop/issues/15956))
+- A few regressions related to the migration of the Customer page:
+- Customer creation with SIRET field ([#16167](https://github.com/PrestaShop/PrestaShop/issues/16167))
```
Comment (matks) [inline news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]:
```suggestion
 - Customer creation with SIRET field ([#16167](https://github.com/PrestaShop/PrestaShop/issues/16167))
```
One more space to make it a 2nd level bullet list (same 2 rows below)

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Important changes
+By Allowing CLDR to display deleted/inactive currencies [#15643](https://github.com/PrestaShop/PrestaShop/pull/15643) we had to add new methods to interfaces `DataSourceInterface`, `CurrencyDataProviderInterface` and `RepositoryInterface` to be able to fetch currencies regardless of them being deleted or inactive. This requires to update any class that implemented that interface.
+
+We had to revert the fix we made for the `object_lang` table issue in 1.7.6.0 regarding multi-store data [#11876](https://github.com/PrestaShop/PrestaShop/pull/11876) as it introduced a regression [#15274](https://github.com/PrestaShop/PrestaShop/issues/15274) that changed how multistore data was persisted. More details here [#15621](https://github.com/PrestaShop/PrestaShop/pull/15621).
+
+
+## Acknowledgments
+
+Contributors to this patch version, from both the Core team and the community at large: 
+Aurélien Pelletier, Jean-François Viguier, Jonathan Lelievre, Khouloud Belguith, Mathieu Ferment, Matthieu Rolland, Pablo Borowicz, Pierre Rambaud, Franck Lefevre, Thomas Baccelli.
```
Comment (matks) [inline news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]: Only us participated to 1.7.6.2 😮 no external contributor ?

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.2"
+subtitle: "Maintenance version of the 1.7.6.x branch"
+date:   2019-11-28 16:15:00
+authors: [ PrestaShop ]
+icon: icon-leaf
```
Comment (sam-pires) [inline news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md:7]: missing `image`

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md (hunk 3 above)
Comment (matks) [reply news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md]: Can we add 202-ecommerce ? PR #16258 was actually a backport of https://github.com/PrestaShop/PrestaShop/pull/16072 so initial credit should go to them

File: news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md (hunk 4 above)
Comment (sam-pires) [inline news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md:7]:
```suggestion
icon: icon-leaf
image: /assets/images/2019/11/meta-1762.png 
```

Comment (ttoine) [reply news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md:7]:
```suggestion
icon: icon-leaf
image: /assets/images/2019/11/meta-1762.png
```

**Comment by ttoine** [comment]
thanks for the writing and the review :-)

**Comment by matks** [comment]
PR https://github.com/PrestaShop/PrestaShop/pull/16258 was a backport of #16072 which is a PR from 202-ecommerce. Can we credit them too ?

PR https://github.com/PrestaShop/PrestaShop/pull/15964 was done following community-reported issue https://github.com/PrestaShop/PrestaShop/issues/14765 including famous prestashop developer prestamodule, maybe we can thank him ?

PR https://github.com/PrestaShop/PrestaShop/pull/15691 was a port of initial PR https://github.com/PrestaShop/PrestaShop/pull/12878 which is a PR from djbuch. Can we credit him too ?

**Comment by ttoine** [comment]
Sure, please feel free to edit the release note and add the credit.
I actually posted what has been given to me by Thomas, Coline and Samuel. I am not sure that they just did know that :-)

**Comment by matks** [comment]
See https://github.com/PrestaShop/prestashop.github.io/pull/603 😉

**Comment by sam-pires** [comment]
Please see #604 as well (updated the meta with Tristan's version)

**Pull request merged**